src/App.js

```diff
@@ -1,10 +1,33 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import data from './data/dummy.json';
 
 const App = () => {
+  const [userAnswers, setUserAnswers] = useState({});
+  const [score, setScore] = useState(0);
+  const [displayScore, shouldDisplayScore] = useState(false);
```

Comment:
The second variable should start with `set`, so this should be `const [shouldDisplayScore, setShouldDisplayScore]`